config/backend-config/types.go

```diff
@@ -52,6 +52,12 @@ type DestinationT struct {
 	Transformations       []TransformationT
 	IsProcessorEnabled    bool
 	RevisionID            string
+	Tunnel                *Tunnelling
+}
+
+type Tunnelling struct {
+	Type   string
+	Config map[string]string
```

Comment:
```suggestion
	Config map[string]interface
```

wdyt?

Reply:
Or if we are completely sure that config backend sends map[string]string then pls ignore this comment.

Reply:
At this moment, to keep the struct fields tight, we are only capturing values as string. Tomorrow in case the DS changes, we will capture it through interface{}